package.json

```diff
@@ -107,7 +107,7 @@
     "husky": "^3.1.0",
     "i18n-js": "http://github.com/fnando/i18n-js/archive/v3.0.0.rc12.tar.gz",
     "identity-obj-proxy": "3.0.0",
-    "jest": "24.9.0",
+    "jest": "26.0.1",
```

Comment:
IMO this should be `^26.1.0`, we've had issue in the past when running with multiple versions of jest. `jest@26.0.1`actually includes "@jest/core@26.1.0". This happens when we install an outdated version.